intellij/src/saros/intellij/editor/EditorManager.java

```diff
@@ -248,9 +248,13 @@ public void resourcesAdded(final IProject project) {
          * transmitted, a second editor activated activity is send for the locally active editor to
          * correctly set the active editor in the remote user editor state for the local user.
          */
-        private void sendAwarenessInformation() {
+        private void sendAwarenessInformation(@NotNull User joinedUser) {
           User localUser = session.getLocalUser();
 
+          if (localUser.equals(joinedUser)) {
```

Comment:
The commit message explains this reasonably well, but I would appreciate a little comment here, stating why we explicitly ignore the `joinedUser` for any uninformed readers of this code.

Reply:
Added explanation to javadoc.